pkg/drivers/hyperkit/driver.go

```diff
@@ -240,6 +245,48 @@ func (d *Driver) Start() error {
 	return nil
 }
 
+func (d *Driver) checkForPidFile() error {
+	stateDir := filepath.Join(d.StorePath, "machines", d.MachineName)
+	pidFile := filepath.Join(stateDir, pidFileName)
+
+	if _, err := os.Stat(pidFile); err != nil {
+		log.Infof("clean start, hyperkit pid file doesn't exist: %s", pidFile)
+		return nil
+	}
+
+	log.Warnf("hyperkit pid file exists: %s", pidFile)
+	// we have a pid file, hyperkit might be running!
+	// Make sure the pid written by hyperkit is the same as in the json
+	content, err := ioutil.ReadFile(pidFile)
+	if err != nil {
+		return errors.Wrapf(err, "reading pidfile %s", pidFile)
+	}
+	pid, err := strconv.Atoi(string(content[:]))
+	if err != nil {
+		return errors.Wrapf(err, "parsing pidfile %s", pidFile)
+	}
+
+	p, _ := os.FindProcess(pid) //noop on linux
```

Comment:
noop comment isn't useful here, since hyperkit is macOS only.

Reply:
ack - copypaste error

Reply:
oh, actually it should be noop on UNIX :)